docker-compose.yml

```diff
@@ -1,18 +1,38 @@
-version: '2'
 services:
   erddap:
-    container_name: "erddap_gold_standard"
     image: axiom/docker-erddap:2.18
     restart: unless-stopped
     ports:
       - "8080:8080"
+    volumes:
+      - ./erddap/conf/robots.txt:/usr/local/tomcat/webapps/ROOT/robots.txt
+      - ./erddap/content:/usr/local/tomcat/content/erddap
+      - ./erddap/data:/erddapData
+      - ./datasets:/datasets
     environment:
+      #note: these can also be set in /usr/local/tomcat/bin/config.sh
       ERDDAP_MIN_MEMORY: 1G
       ERDDAP_MAX_MEMORY: 2G
-    volumes:
-      - "${PWD}/erddap/conf/config.sh:/usr/local/tomcat/bin/config.sh"
-      - "${PWD}/erddap/conf/robots.txt:/usr/local/tomcat/webapps/ROOT/robots.txt"
-      - "${PWD}/erddap/content:/usr/local/tomcat/content/erddap"
-      - "${PWD}/erddap/data:/erddapData"
-      - "${PWD}/datasets:/datasets"
-      - "/tmp/:/usr/local/tomcat/temp/"
+      ERDDAP_bigParentDirectory: /erddapData/
+      ERDDAP_baseUrl: http://localhost:8080
+      ERDDAP_baseHttpsUrl: https://localhost:8080
+      ERDDAP_flagKeyKey: change-this-value!!
+      ERDDAP_emailEverythingT: this+goes+nowhere@doesnotexist.ioos.us
+      ERDDAP_emailDailyReportsTo:
+      ERDDAP_emailFromAddress:
+      ERDDAP_emailUserName:
+      ERDDAP_emailPassword:
+      ERDDAP_emailProperties:
+      ERDDAP_emailSmtpHost:
+      ERDDAP_emailSmtpPort:
+      ERDDAP_adminInstitution: Integrated Ocean Observing System (IOOS)
+      ERDDAP_adminInstitutionUrl: https://ioos.github.io/
+      ERDDAP_adminIndividualName: Micah Wengren
```

Comment:
I'm not sure how my contact info made it in here originally, but let's please use something generic instead:

Individual name: IOOS Data Manager
Phone: 240-533-9444
Email: data.ioos@noaa.gov

Thanks!

Reply:
Done!